drivers/pinctrl/Kconfig

```diff
@@ -34,7 +34,9 @@ config PINCTRL_DYNAMIC
 	  peripheral at early boot stages depending on a certain input.
 
 source "drivers/pinctrl/Kconfig.b91"
+source "drivers/pinctrl/Kconfig.bflb"
 source "drivers/pinctrl/Kconfig.ambiq"
+source "drivers/pinctrl/Kconfig.bflb"
```

Comment:
`Kconfig.bflb` is being included two times

Reply:
Fixed